packages/xo-web/src/xo-app/home/index.js

```diff
@@ -697,7 +703,7 @@ export default class Home extends Component {
     return event => setFilter(event.target.value)
   })
 
-  _getFilteredItems = createSort(
+  __getFilteredItems = createSort(
```

Comment:
Maybe find a better name? There are 2 selectors with the same name and it's hard to know which one does what without reading the code.

Reply:
You can probably inline it if you only use it once. Except if you wanted to keep it separate for readability.